src/pasteup/grid.js

```diff
@@ -47,11 +47,14 @@ export const calculateWidth = (breakpoint, colspan) => {
     );
 };
 
-const gridStyles = (breakpoint, [colspan]) => ({
+const gridStyles = (breakpoint, [colspan, options]) => ({
     [breakpointMqs[breakpoint]]: {
         float: 'left',
         width: calculateWidth(breakpoint, colspan) + gutter,
         paddingLeft: gutter,
+        marginLeft: options.inset
```

Comment:
how will this work if there's a col before it? i guess you have manually account for that?

Reply:
Sorry, can you expand on that?

Reply:
sorry - if you had:

```js
<Row>
  <Cols wide={[2, { inset: 2 }]}> 
    <Component.a />
  </Cols>
  <Cols wide={[2, { inset: 1 }]}>
    <Component.b />
  </Cols>
</Row>
```
would it look like □□■■□■■□□□□□□□□□?

so that inset is the number of columns in from where it would naturally appear, rather than from  full left (column 0)?